activestorage/test/models/variant_test.rb

```diff
@@ -275,11 +275,19 @@ class ActiveStorage::VariantTest < ActiveSupport::TestCase
 
   private
     def process_variants_with(processor)
-      previous_processor, ActiveStorage.variant_processor = ActiveStorage.variant_processor, processor
+      previous_transformer = ActiveStorage.variant_transformer
+      ActiveStorage.variant_transformer =
+        case processor
+        when :vips
+          ActiveStorage::Transformers::Vips
+        when :mini_magick
+          ActiveStorage::Transformers::ImageMagick
+        end
```

Comment:
Might want to have an `else` that raises if the argument wasn't any of the two, this way if someone typo `:mini_magick` into say `:minimagick`, they get an early error.